standalone/src/models/product-selection/product-selection-draft/fields-config.ts

```diff
@@ -0,0 +1,25 @@
+import { fake, TModelFieldsConfig } from '../../../core';
+import {
+  TProductSelectionDraftGraphql,
+  TProductSelectionDraftRest,
+} from '../types';
+
+export const restFieldsConfig: TModelFieldsConfig<TProductSelectionDraftRest> =
+  {
+    fields: {
+      name: fake((f) => f.lorem.word()),
+      key: fake((f) => f.lorem.slug()),
+      custom: null,
+      mode: 'Individual',
+    },
+  };
+
+export const graphqlFieldsConfig: TModelFieldsConfig<TProductSelectionDraftGraphql> =
+  {
+    fields: {
+      name: fake((f) => f.lorem.word()),
+      key: fake((f) => f.lorem.slug()),
+      custom: null,
+      mode: 'Individual',
```

Comment:
`name` property seem to have different share in the GraphQL type so we need to adjust here accordingly.